app/models/post.rb

```diff
@@ -13,7 +13,9 @@ class Post < ActiveRecord::Base
   after_create :notify_twitter
   # TODO: move this to background job
   def notify_twitter
-    $twitter_client.update(tweet_content)
+    if not Rails.env.development? and not Rails.env.test?
```

Comment:
You can change this to say `if Rails.env.production?` because it is the only one where this should send.